src/annotator/util/frame-util.js

```diff
@@ -15,29 +15,43 @@ export function findFrames(container) {
   return Array.from(container.querySelectorAll('iframe[enable-annotation]'));
 }
 
-// Check if the iframe has already been injected
+/**
+ * Check if the iframe has already been injected
+ *
+ * * @param {HTMLIFrameElement} iframe
+ */
 export function hasHypothesis(iframe) {
   return '__hypothesis' in iframe.contentWindow;
 }
 
-// Inject embed.js into the iframe
-export function injectHypothesis(iframe, scriptUrl, config) {
+/**
+ * Inject embed.js into the iframe
+ *
```

Comment:
I suggest adding an additional comment here that `iframe` must be same-origin with the current window. If we document this as a requirement of the function then the implementation could handle the `null` case for `contentDocument` by just throwing an error.

Reply:
I have added a comment.

I have done some testing to check if `iframe.contentDocument` can return `null` from same-origin iframes.  I checked in latest version of Firefox, Safari and Chrome, and `iframe.contentDocument` always returned a `Document` (even if it was an initial `about:blank`).

'injectHypothesis` effectively doesn't throw an error because it is used after `isAccessible` function , that checks the frame is from the same origin and hence accesible.